conda-nighres.yml

```diff
@@ -1,36 +1,10 @@
 name: nighres
 channels:
-  - mrtrix3
-  - simpleitk
-  - itk
   - conda-forge
-  - bioconda
-  - defaults
 dependencies:
   - python=3.9
   - pip
-  - numpy
-  - nibabel
-  - psutil
   - jcc
-  - pandas
-  - pyyaml
-  - scikit-image
-  - scikit-learn
   - Nilearn
-  - statsmodels
-  - matplotlib
-  - Pillow
-  - plotly
-  - charset-normalizer
-  - chart-studio
-  - idna
-  - requests
-  - retrying
-  - urllib3
-  - tenacity
-  - webcolors
```

Comment:
All but of those but `jcc` and `Nilearn` will be installed as dependencies of `antspyx` from the `setup.py`

Reply:
> Yup I know but I don't see the point of putting listing them here if they are going to be installed as dependencies of something else.

They will be installed differently.

> Or in that case let's be exhaustive and do the conda equivalent of a "pip freeze".

I'm okay with that for "release" versions of nighres, for reproducability.

Reply:
Okidoki. Will fix that.